component/src/main/java/org/wso2/extension/siddhi/io/websocket/util/WebSocketConstants.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.io.websocket.util;
+
+/**
+ * This class represents WebSocket specific Constants.
+ */
+public class WebSocketConstants {
+    private WebSocketConstants() {
+    }
+
+    public static final String URL = "url";
+    public static final String HEADERS = "headers";
+    public static final String SUB_PROTOCOL = "sub.protocol";
+    public static final String IDLE_TIMEOUT = "idle.timeout";
+    public static final String SUB_PROTOCOL_SPLITTER = ",";
```

Comment:
Let's move these constants to where they are use as `private static final` constants